action/rlp_tx.go

```diff
@@ -93,6 +89,8 @@ func NewEthSigner(txType iotextypes.Encoding, chainID uint32) (types.Signer, err
 		return types.NewEIP155Signer(big.NewInt(int64(chainID))), nil
 	case iotextypes.Encoding_ETHEREUM_UNPROTECTED:
 		return types.HomesteadSigner{}, nil
+	case iotextypes.Encoding_ETHEREUM_ACCESSLIST:
+		return types.NewEIP2930Signer(big.NewInt(int64(chainID))), nil
```

Comment:
is EIP2930Signer compatible with EIP155Signer?

Reply:
no, it's not

Reply:
per offline discussion, `eip2930signer` is backward-compatible with `eip155signer`, so let's follow go-ethereum's programming and use the new/latest signer